### PR TITLE
[tests-only][full-ci] adding user quota using graph api instead of Provisioning api

### PR DESCRIPTION
**Pull request by S-Panta**
## Description
This PR changes the test code and feature lines where the setting of user quota uses graph API. Previously, the test wasn't passing as it used provisioning API but at the moment oCIS uses graph for that task. 

## Related Issue

- Part of 
- https://github.com/owncloud/ocis/issues/6729#event-9985326651
- https://github.com/owncloud/ocis/issues/6998

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by SwikritiT** [comment]
@S-Panta what is the status of this PR/ It's been a long time can we get this merged? or closed if it is not relevant anymore?

**Comment by S-Panta** [comment]
I will continue on this PR next week.

**Comment by S-Panta** [comment]
I feel that the scenario of the feature file `getQuota` inside the `coreWebDavProperties1` should only be limited to Reva. Setting and Getting the quota using graphs and ocs ( in Reva) have different sets of request and response bodies. For example, for a graph API, the quota should be given as an integer while the ocs API uses `10 MB` as the body to set any quota. The response seems different too. 
Should this PR be closed but separate issues be opened so that tests could be added regarding getting quota as I find no such test in other files inside apiGraph? @saw-jan @SwikritiT

**Comment by S-Panta** [comment]
I have refactored this file so that the test coverage for the graph related to user quota is covered.

File: tests/acceptance/features/coreApiWebdavProperties1/getQuota.feature [hunk 1]
```diff
@@ -23,9 +23,9 @@ Feature: get quota
   @smokeTest
   Scenario Outline: retrieving folder quota when quota is set
     Given using <dav-path-version> DAV path
-    When the administrator sets the quota of user "Alice" to "10 MB" using the provisioning API
+    When the administrator sets the quota of user "Alice" to "10000" using the provisioning API
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavProperties1/getQuota.feature:26]: let's use these steps:
- `Given user "Admin" has changed the quota of the "Alice Hansen" space to "10000"`
- `When user "Admin" changes the quota of the "Alice Hansen" space to "10000" owned by user "Alice"`

and remove provisioning ones (if can be). There will be confusion if we have multiple similar steps

File: tests/acceptance/features/coreApiShareOperationsToShares2/uploadToShare.feature [hunk 2]
```diff
@@ -165,14 +165,14 @@ Feature: sharing
   @skipOnGraph
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareOperationsToShares2/uploadToShare.feature:165]: Could you check these tags?
```suggestion
```

Comment (S-Panta) [reply tests/acceptance/features/coreApiShareOperationsToShares2/uploadToShare.feature:165]: The PR related to uploading file on share is failing and thus even if the quota code is updated to graph, this doesn't work.

Comment (saw-jan) [reply tests/acceptance/features/coreApiShareOperationsToShares2/uploadToShare.feature:165]: then we have to fix it or remove the test scenario if it's not valid.

**Comment by S-Panta** [comment]
Solve this PR Only after this issue is solved. 
https://github.com/owncloud/ocis/issues/8197

File: tests/acceptance/features/coreApiWebdavProperties1/getQuota.feature [hunk 3]
```diff
@@ -36,53 +36,53 @@ Feature: get quota
...
       | permissions | all       |
       | shareWith   | Alice     |
-    When user "Alice" gets the following properties of folder "/testquota" using the WebDAV API
+    When user "Alice" gets the following properties of folder "<folder-path>" inside space "Shares" using the WebDAV API
       | propertyName            |
       | d:quota-available-bytes |
-    Then the HTTP status code should be "200"
-    And the single response should contain a property "d:quota-available-bytes" with value "10485406"
+    Then the HTTP status code should be "207"
+    And the single response should contain a property "d:quota-available-bytes" with value "10000"
```
[the remark this answers is not in the document]
Comment (S-Panta) [reply tests/acceptance/features/coreApiWebdavProperties1/getQuota.feature:51]: `d:quota-available-bytes` is responding wrong value. In practice, it should show the quota of Brian that is available for that folder not the full quota. Even when other files are inside Brian , the propfind would show all the remaining spaces of Brian. So maybe this is related to the issue too.

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 4]
```diff
@@ -144,11 +148,6 @@ cannot share a folder with create permission
 - [coreApiTrashbin/trashbinSharingToShares.feature:201](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiTrashbin/trashbinSharingToShares.feature#L201)
 - [coreApiTrashbin/trashbinSharingToShares.feature:224](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiTrashbin/trashbinSharingToShares.feature#L224)
 
-#### [changing user quota gives ocs status 103 / Cannot set quota](https://github.com/owncloud/product/issues/247)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:147]: Please, take care of this issue

**Review by saw-jan — APPROVED** [review]
LGTM

**Pull request merged**